libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -31,7 +32,11 @@
 #endif
 
 #if CONFIG_HAL_BOARD == HAL_BOARD_PX4
-# define BOARD_SAFETY_ENABLE_DEFAULT 1
+# if APM_BUILD_TYPE(APM_BUILD_ArduSub)
+#  define BOARD_SAFETY_ENABLE_DEFAULT 0
+# else
+#  define BOARD_SAFETY_ENABLE_DEFAULT 1
```

Comment:
We keep the electronics in a sealed tube, it would be quite an ordeal to disassemble/reassemble to access the switch. We could address this in our default parameter file instead.